File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        set<int> mySet;
        for(int n : nums){
            if(mySet.count(n) > 0){
                return true;
            }else{
                mySet.insert(n);
            }
        }
        return false;
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        mySet = set()
        l = len(nums)
        for i in range(l):
            if nums[i] in mySet:
                return True
            else:
                mySet.add(nums[i])
        return False